### PR TITLE
elastic7 support

### PR DESCRIPTION
**Pull request by petrjasek**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+            - "6379:6379"
+
+    mongo:
+        image: mongo:3.4
+        ports:
+            - "27017:27017"
+        tmpfs:
+            - /data/db
+
+    rabbitmq:
```
Comment (ride90) [inline docker-compose.yml]: Why do we need a rabbitmq ?

Comment (petrjasek) [reply docker-compose.yml]: we don't, I just wanted to run tests in environment close to production

Comment (petrjasek) [reply docker-compose.yml]: but it's probably not used anyhow due to eager celery, so will remove it

File: superdesk/metadata/item.py [hunk 2]
```diff
@@ -231,11 +231,13 @@
         'type': 'string',
         'mapping': {
             'type': 'string',
+            'fielddata': True,
```
Comment (ride90) [inline superdesk/metadata/item.py]: Why do we need `fielddata` here? For sorting?

File: superdesk/publish/publish_content.py [hunk 3]
```diff
@@ -157,7 +157,11 @@ def transmit_item(queue_item_id, is_async=False):
         logger.info('Transmitting queue item {}'.format(log_msg))
 
         destination = queue_item['destination']
-        transmitter = superdesk.publish.registered_transmitters[destination.get('delivery_type')]
+        try:
+            transmitter = superdesk.publish.registered_transmitters[destination.get('delivery_type')]
+        except KeyError:
+            print(destination['delivery_type'] not in superdesk.publish.registered_transmitters)
```
Comment (ride90) [inline superdesk/publish/publish_content.py]: Forgot to remove a print statement.

Comment (ride90) [reply superdesk/publish/publish_content.py]: I think having a `logger.exception/error` might be useful here.

**Comment by petrjasek** [comment]
it's changed [there in this PR](https://github.com/superdesk/superdesk-core/pull/1738/files#diff-5d4eecebe8590ab62714759eb3959404R52)

**Pull request merged**